src/features/MentorPage/MentorPage.tsx

```diff
@@ -527,16 +528,41 @@ const mentorListCards: Array<ListCardProps> = mentorCards.map(item => {
   return cardProps;
 });
 
+export type handleSetVisibleCardProps = {
+  shouldShowMentorCard: boolean;
+  mentorCardData: ListCardProps;
+};
+
+let currentCard: ListCardProps;
+
 const MentorPage = () => {
-  const [shouldShowMentorCard, setVisibleCard] = React.useState<{
-    [key: string]: unknown;
-  }>({});
+  const [shouldShowMentorCard, setVisibleCard] = React.useState(false);
 
-  const handleSetVisibleCard = (mentorCardData: ListCardProps) => {
-    setVisibleCard(mentorCardData);
-    console.log('shouldShowVisibleCard:', shouldShowMentorCard);
+  const handleSetVisibleCard = ({
+    shouldShowMentorCard,
+    mentorCardData,
+  }: handleSetVisibleCardProps) => {
+    setVisibleCard(shouldShowMentorCard);
+    currentCard = mentorCardData;
   };
-
+  if (shouldShowMentorCard) {
+    return (
+      <PageLayout>
+        <MentorCard
+          setVisibleCard={handleSetVisibleCard}
+          mentorCardData={currentCard}
+        />
+        <OneContainerLayout headLine={mentorPageHeadline}>
+          <MentorInfoSearchDiv />
+          <MentorChips items={mentorChipList} />
+        </OneContainerLayout>
+        <MentorListItems
+          setVisibleCard={handleSetVisibleCard}
+          mentorCardData={mentorListCards}
+        />
+      </PageLayout>
+    );
+  }
```

Comment:
Here we have defined again a _let currentCard_, instead use useState for it.
In the return we have two almost similar renders, we can simplify it.

With the two changes we could do it like this:
![image](https://user-images.githubusercontent.com/34128180/183021725-aa79be0e-f985-4438-b4f1-def530a58a16.png)

Reply:
Also: we could refactor this a bit.
 Idea: Only use currentCard for tracking the status if the Card should be open or not. 
 
 ```javascript
 type CurrentCard = 'asd' | undefined
 
 const [currentCard, setCurrentCard] = useState<CurrentCard>()
 
return ( currentCard && <div>Lolwut</div> )
 ```
 
 This will evualuate to false and div is not shown - So we could get rid of the other variable =P